components/ILIAS/Authentication/classes/class.ilSessionReminder.php

```diff
@@ -104,13 +143,11 @@ private function isEnoughTimeLeftForReminder(): bool
 
     public function isActive(): bool
     {
-        return (
-            self::isGloballyActivated() &&
+        return
             !$this->getUser()->isAnonymous() &&
             $this->getUser()->getId() > 0 &&
-            (int) $this->getUser()->getPref('session_reminder_enabled') &&
-            $this->isEnoughTimeLeftForReminder()
-        );
+            $this->isIndividuallySet() &&
```

Comment:
Why not `$this->getEffectiveLeadTime() > 0`? We could then remove `isIndividuallySet`.